### DISCUSSION TITLE
Able to edit size?

### DISCUSSION THREAD

**Discussion by denveroh** [lezzthanthree/Needy-Streamer-Overload#25] [General]
I want ame webcam to be larger on my 28 inch monitor
same with Jine >.<

Also thank you for making this!

**Comment by lezzthanthree** [comment]
Hey there! I just made an update where you can [resize Ame-chan](https://github.com/lezzthanthree/Needy-Streamer-Overload/releases/tag/v.5.2)! You better update now~

I'll be planning JINE soon~

**Reply by denveroh** [reply]
Omg Bless!!
